include/deal.II/numerics/data_postprocessor.h

```diff
@@ -816,6 +820,245 @@ class DataPostprocessorVector : public DataPostprocessor<dim>
+ *                               DataOut<dim>::type_dof_data,
+ *                               data_component_interpretation);
+ *     data_out.add_data_vector (solution, grad_u);
+ *     data_out.build_patches ();
+ *     data_out.write_vtk (output);
+ * @endcode
+ *
+ * This leads to the following output for the displacement field (i.e., the
+ * solution) and the gradients (you may want to compare with the solution shown
+ * in the results section of step-8; the current data is generated on a uniform
+ * mesh for simplicity):
+ *
+ * @image html data_postprocessor_tensor_0.png
+ * @image html data_postprocessor_tensor_1.png
+ *
+ * These pictures show an ellipse representing the gradient tensor at, on average,
+ * every tenth mesh point. You may want to read through the documentation of the
+ * Visit visualization program (see https://wci.llnl.gov/simulation/computer-codes/visit/)
+ * for an interpretation of how exactly tensors are visualizated.
+ *
+ * In elasticity, one is often interested not in the gradient of the displacement,
+ * but in the "strain", i.e., the symmetrized version of the gradient
+ * $\varepsilon=\frac 12 (\nabla u + \nabla u^T)$. This is easily facilitated with
+ * the following minor modification:
+ * @code
+ *   template <int dim>
+ *   class StrainPostprocessor : public DataPostprocessorTensor<dim>
+ *   {
+ *   public:
+ *     StrainPostprocessor ()
+ *       :
+ *       DataPostprocessorTensor<dim> ("strain",
+ *                                     update_gradients)
+ *     {}
+ *
+ *     virtual
+ *     void
+ *     evaluate_vector_field (const DataPostprocessorInputs::Vector<dim> &input_data,
+ *                            std::vector<Vector<double> >               &computed_quantities) const
+ *     {
+ *       AssertDimension (input_data.solution_gradients.size(),
+ *                        computed_quantities.size());
+ *
+ *       for (unsigned int p=0; p<input_data.solution_gradients.size(); ++p)
+ *         {
+ *           AssertDimension (computed_quantities[p].size(),
+ *                            (Tensor<2,dim>::n_independent_components));
+ *           for (unsigned int d=0; d<dim; ++d)
+ *             for (unsigned int e=0; e<dim; ++e)
+ *               computed_quantities[p][Tensor<2,dim>::component_to_unrolled_index(TableIndices<2>(d,e))]
+ *                 = (input_data.solution_gradients[p][d][e]
+ *                    +
+ *                    input_data.solution_gradients[p][e][d]) / 2;
+ *         }
+ *     }
+ *   };
+ * @endcode
+ *
+ * Using this class in in step-8 leads to the following visualization:
+ *
+ * @image html data_postprocessor_tensor_2.png
+ *
+ * Given how easy it is to output the strain, it would also not be very
+ * complicated to write a postprocessor that computes the <i>stress</i>
+ * in the solution field as the stress is easily computed from the
+ * strain by multiplication with either the strain-stress tensor or,
+ * in simple cases, the Lame constants.
```

Comment:
"Lamé"

Reply:
Do you know how to do this in doxygen?

Reply:
Maybe with a html unicode character. I think its [`&eacute`](http://www.ascii.cl/htmlcodes.htm)?